package.json

```diff
@@ -22,5 +22,5 @@
         "sass-loader": "^7.1.0",
         "vue": "^2.5.17",
         "vue-template-compiler": "^2.5.21"
-    }
-}
+    },
```

Comment:
JSON standard doesn't allow trailing comma. It trows an error. Please remove it.